modules/core/src/test/java/org/apache/ignite/internal/GridNodeMetricsLogSelfTest.java

```diff
@@ -85,12 +108,12 @@ public void testNodeMetricsLog() throws Exception {
         IgniteCache<Integer, String> cache1 = grid(0).createCache("TestCache1");
         IgniteCache<Integer, String> cache2 = grid(1).createCache(
             new CacheConfiguration<Integer, String>("TestCache2")
-            .setDataRegionName(persistenceEnabled() ? IN_MEMORY_REGION : "default"));
+                .setDataRegionName(persistenceEnabled() ? IN_MEMORY_REGION : "default"));
 
         cache1.put(1, "one");
         cache2.put(2, "two");
 
-        Thread.sleep(10_000);
+        Thread.sleep(5_000);
```

Comment:
`Thread.sleep()` looks weird to me. What kind of event do we wait for? Perhaps, we should use `waitForCondition()`. WDYT?